### DISCUSSION TITLE
DIAMOND v2.0.15

### DISCUSSION THREAD

**Discussion by bbuchfink** [bbuchfink/diamond#578] [General]
- Fixed a bug (present since v2.0.12) that caused the `diamond view` workflow to report a zero bit score for all alignments.

<hr /><em>This discussion was created from the release <a href='https://github.com/bbuchfink/diamond/releases/tag/v2.0.15'>DIAMOND v2.0.15</a>.</em>

**Comment by amrhemeda** [comment]
How I can download this on MacBook? I need version for my macbook

**Reply by bbuchfink** [reply]
You can try to download using bioconda, homebrew or compiling from source.

**Reply by jedsoncardoso** [reply]
/data/database/diamond/src/output/daa/daa_write.cpp: In function ‘void finish_daa(OutputFile&, DAA_file&, const StringSet&, const std::vector<unsigned int>&, int64_t)’:
/data/database/diamond/src/output/daa/daa_write.cpp:146:24: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
  for (int64_t i = 0; i < seq_ids.size(); ++i) {

**Comment by jedsoncardoso** [comment]
When performing analysis using NR NCBI database, it takes a long time to write DAA file, my dataset has 1162522 reads only. My server has 80 threads and 1TB of RAM. However, before to DIAMOND v2.0.15., it was performed in 12 hours. I think that new version has some error?

**Reply by bbuchfink** [reply]
> When performing analysis using NR NCBI database, it takes a long time to write DAA file, my dataset has 1162522 reads only. My server has 80 threads and 1TB of RAM. However, before to DIAMOND v2.0.15., it was performed in 12 hours. I think that new version has some error?

I tested this and didn't see a performance problem writing DAA files. Please let me know the nature of your input data and your command line parameters.

**Reply by jedsoncardoso** [reply]
#diamond v2.0.15.153
#Sample are Paired-end Reads
diamond blastx -d nr_x86 -p $4 -q Sample_merged.fastq -o Sample_merged_nr_diamond -t Sample_temp_dir -f 100 -k 1 -b 18 -c 1 --salltitles -e 0.0001 --unal 1 --un Sample_merged_nr_diamond_unaligned

**Reply by jedsoncardoso** [reply]
#diamond v2.0.15.153
#Sample are Paired-end Reads
diamond blastx -d nr_x86 -p 80 -q Sample_merged.fastq -o Sample_merged_nr_diamond -t Sample_temp_dir -f 100 -k 1 -b 18 -c 1 --salltitles -e 0.0001 --unal 1 --un Sample_merged_nr_diamond_unaligned

**Reply by jedsoncardoso** [reply]
Dear,
I downgraded Diamond to v2.0.11.149, re-indexed NCBI
 NR Database. Then, I performed the analysis of the 1,160,907 reads again.
=============================================
This script started at Tue Oct 4 19:04:23 -03 2022
This script finished at Wed Oct 5 02:14:02 -03 2022
=============================================
DIAMOND_DB=/data/database/DiamondDB

/opt/02_Alignment/diamond/bin/diamond blastx -d $DIAMOND_DB/nr_x86 -p 40 -q Sample_merged.fastq -o Sample_merged_nr_diamond -t Sample_temp_dir -f 100 -k 1 -b 18 -c 1 --salltitles -e 0.0001 --unal 1 --un Sample_merged_nr_diamond_unaligned

**Comment by learnitXD** [comment]
After downloading the Windows.zip which has the diamond application, 
I double clicked , then select run or install, but no windows came up.
How can I install Diamond to Windows 11?

**Reply by bbuchfink** [reply]
You need to run diamond from the command prompt.

**Reply by wandersonlimacunha1** [reply]
This didn't work for me, start using "start diamond.exe", and it just disappeared

**Reply by DrSHI2024** [reply]
I just encountered the same problem in windows11.

**Reply by bbuchfink** [reply]
Using `start` is not correct. You just need to open the command prompt and run `diamond`.
